### DISCUSSION TITLE
Search Function?

### DISCUSSION THREAD

**Discussion by jfdid** [hargata/lubelog#436] [Ideas]
Hi folks - amazing project, I donated today.
I'm wondering if there's a way to add a global search(within each vehicle profile) for repairs/service records?  For example, if I wanted to find a keyword "brakes" within the repairs/service records.
Thanks for your consideration!

**Comment by hargata** [comment]
Thanks for the donation! I have started looking into this, is it okay if it just searches through the "Description", "Tags" and "Notes" for the keyword for now? This is for performance reasons(it greatly slows down once we try searching for the keyword in more fields)

I have created #448 to track this.

Once again, thank you!

**Comment by jfdid** [comment]
This is awesome!  Thanks very much.  Maybe in the future, the Case Sensitive could be removed, but this is definitely a working solution.
